source/extensions/common/wasm/wasm.cc

```diff
@@ -1593,10 +1593,16 @@ uint32_t Context::httpCall(absl::string_view cluster, const Pairs& request_heade
     token = next_http_call_token_++;
   }
   auto& handler = http_request_[token];
+
+  // set default hash policy to be based on :authority to enable consistent hash
+  Http::AsyncClient::RequestOptions options;
+  options.setTimeout(timeout);
+  Protobuf::RepeatedPtrField<envoy::api::v2::route::RouteAction::HashPolicy> hash_policy;
+  hash_policy.Add()->mutable_header()->set_header_name(Http::Headers::get().Host.get());
```

Comment:
If this is the accessor you are going to use then please change the comment to be :authority/Host.

Reply:
Updated the PR title and the comment above. Envoy conflates host and :authority, they are actually the same in the code.